jobs/mongodb_migration/src/mongodb_migration/config.py

```diff
@@ -6,42 +6,37 @@
 from environs import Env
 from libcommon.config import CacheConfig, CommonConfig, QueueConfig
 
-from mongodb_migration.database_migrations import connect_to_database
-
 MONGODB_MIGRATION_MONGO_DATABASE = "datasets_server_maintenance"
-MONGO_DATABASE_MONGO_URL = "mongodb://localhost:27017"
+MONGODB_MIGRATION_MONGO_URL = "mongodb://localhost:27017"
```

Comment:
According to the defined class at line 14, It might be DATABASE_MIGRATION_MONGO_URL value
And also attribute name at line 9 could be DATABASE_MIGRATION_MONGO_DATABASE
Just to keep the consistency between NAMESPACE_ -> Class <Namespace>Config as it is done in other configurations classes

Reply:
Fixed, thanks